modules/javafx.base/src/main/java/javafx/util/converter/CurrencyStringConverter.java

```diff
@@ -42,30 +41,45 @@
  */
 public class CurrencyStringConverter extends NumberStringConverter {
 
-    // ------------------------------------------------------------ Constructors
+    /**
+     * Constructs a {@code CurrencyStringConverter} with the default locale and format.
+     */
     public CurrencyStringConverter() {
         this(Locale.getDefault());
     }
 
+    /**
+     * Constructs a {@code CurrencyStringConverter} with the given locale and the default format.
```

Comment:
Can you also add the `@param locale` tag? This applies to all of the newly documented constructors that have arguments.